corehq/apps/locations/tasks.py

```diff
@@ -172,3 +172,11 @@ def update_users_at_locations(domain, location_ids, supply_point_ids, ancestor_i
     # update fixtures for users at ancestor locations
     user_ids = user_ids_at_locations(ancestor_ids)
     update_fixture_status_for_users(user_ids, UserFixtureType.LOCATION)
+
+
+def deactivate_users_at_location(location_id):
```

Comment:
I think this should also probably move to `custom.icds`, though I feel more strongly about `deprecate_by`.

Reply:
hmm, i think this is a fine place for it since its actually bound to locations only, irrespective of project. no?

Reply:
Probably depends on whether or not this is an ICDS-only feature or will potentially be useful for other projects, pinged you on slack to chat about that.